app/views/groups/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="splash header">
+  <%= image_tag("menu.png", alt: "Menu icon") %>
+  <h2>CATEGORIES</h2>
+  <%= image_tag("search.png", alt: "Search icon") %>
+</div>
+
+<ul>
+  <% @groups.each do |group| %>
+    <li class="group">
+      <%= link_to group_path(group) do %>
+        <% if group.icon.present? %>
+          <%= image_tag group.icon %>
+        <% else %>
+          <%= image_tag asset_path('under-construction.jpeg') %>
+        <% end %>
+        <h2><%= group.name %></h2>
+        <p>$<%= group.movements.sum(:amount) %></p>
+      <% end %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- Kindly improve the design, Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [x] Kindly follow the provided design (width, font, size, height, background, color...)
1. Place the "NEW CATEGORY" button at the bottom of your page. ⬇️ 
2. The transaction button has a similar problem. Kindly fix them as well. 💯 👍🏻 
3. The total amount number should be bold and not normal text, Please increase this font width. 👍🏻 
4. The category name text should not be in bold, let's make it normal text width. ✍🏻 
5. The card should have a category image, name, date, and amount. kindly let's display the create date for each category. 🛀🏻 

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/228997493-71d87801-7d72-4902-a9f1-53d1e77361cd.png) | ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)